### PR TITLE
added a tool name constants file

### PR DESCRIPTION
**Pull request by qianz2**
- [ ] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
- [ ] Tests are included and/or updated for code changes.
- [ ] Proper license headers are included in each file.

### CODE REVIEW

File: src/BuildScriptGenerator/ToolNameConstants.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    /// </summary>
+    public static class ToolNameConstants
+    {
+        public const string PythonName = "python";
+
+        public const string PhpName = "php";
+
+        public const string DotnetName = "dotnet";
+
+        public const string NodeToolName = "node";
```
Comment (gpcastro) [inline src/BuildScriptGenerator/ToolNameConstants.cs]: nit: remove `Tool` to match the other values

File: src/BuildScriptGenerator/ToolNameConstants.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    /// <summary>
+    /// Represents tool names for different platforms.
+    /// </summary>
+    public static class ToolNameConstants
+    {
+        public const string PythonName = "python";
+
+        public const string PhpName = "php";
+
+        public const string DotnetName = "dotnet";
```
Comment (dorfire) [inline src/BuildScriptGenerator/ToolNameConstants.cs]: capitalize the N (DotNet)

File: src/BuildScriptGenerator/ToolNameConstants.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+namespace Microsoft.Oryx.BuildScriptGenerator
+{
+    /// <summary>
+    /// Represents tool names for different platforms.
+    /// </summary>
+    public static class ToolNameConstants
```
Comment (arroyc) [inline src/BuildScriptGenerator/ToolNameConstants.cs]: nit: PlatformToolNameConstants?

File: src/BuildScriptGenerator/ToolNameConstants.cs [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    {
+        public const string PythonName = "python";
```
Comment (arroyc) [inline src/BuildScriptGenerator/ToolNameConstants.cs]: nit:PlatformPython or just Python??

File: src/BuildScriptGenerator/ToolNameConstants.cs (hunk 3 above)
Comment (qianz2) [reply src/BuildScriptGenerator/ToolNameConstants.cs]: Not sure which is more suitable, PlatformTooNamesConstants should be great.

Comment (dorfire) [reply src/BuildScriptGenerator/ToolNameConstants.cs]: i personally like PlatformToolNames; If we want, we can shove it in a Constants namespace to make it clearer that it contains constants

**Pull request merged**